python/cudf/cudf/core/column/timedelta.py

```diff
@@ -227,6 +227,20 @@ def _binary_op_truediv(
 
         return lhs, rhs, out_dtype
 
+    def _make_copy_with_na_as_null(self):
```

Comment:
Is it appropriate for this to be copy-pasted from `DatetimeColumn`? I would consider making this a standalone function that is called in `column.py` where needed instead of having this method defined on both classes.

There are only three invocations of this function which all follow the pattern `col = col._make_copy_with_na_as_null()`. It seems like a free function would be better: `col = _make_copy_with_na_as_null(col)`.